pkg/apis/openebs.io/v1alpha1/cstor_volume_claim.go

```diff
@@ -56,6 +56,8 @@ type CStorVolumeClaimSpec struct {
 	// CStorVolumeRef has the information about where CstorVolumeClaim
 	// is created from.
 	CStorVolumeRef *corev1.ObjectReference `json:"cstorVolumeRef,omitempty"`
+	// CstorVolumeSource contains the source volumeID+Snapshot combaination
```

Comment:
nit: Should we have something like `volumeName@snapShotname` and this will be filled only if it is a clone creation